### PR TITLE
implement support page

### LINKED ISSUE #57: Implement Support Page
image

- Mascot should be in the public folder as an image
- Use Title.jsx component and pass in the food and color prop.

### PR DESCRIPTION
**Pull request by xwRongx**
image

### CODE REVIEW

**Review by qhgill — DISMISSED** [review]
add background color to the support page to match Figma, otherwise looks good!

File: src/components/support/Support.jsx [hunk 1]
```diff
@@ -2,15 +2,22 @@ import Connect from "@/components/support/Connect";
...
+      <div className="grid grid-cols-2 items-center justify-center py-14">
+        <Image src={standing} alt="standing" />
+        <div className="flex flex-col items-center gap-y-8">
+          <Subtitle color="text-biscuits-brown-200" text="Donations" />
+          <Button text="VENMO" link="https://account.venmo.com/u/h4b_ucr" />
+          <Subtitle color="text-biscuits-brown-200" text="Newsletter" />
+          <Button text="SIGN UP" link="http://eepurl.com/iaWAc5" />
+        </div>
       </div>
       <div className="text-center text-biscuits-brown-200 font-shrikhand text-5xl">
```
Comment (qhgill) [inline src/components/support/Support.jsx]: replace this with subtitle component

File: src/components/support/Support.jsx [hunk 2]
```diff
@@ -2,15 +2,22 @@ import Connect from "@/components/support/Connect";
...
-    <div>
-      <div className="flex flex-col items-center gap-y-8">
-        <Subtitle color="text-biscuits-brown-200" text="Donations" />
-        <Button text="VENMO" link="https://account.venmo.com/u/h4b_ucr" />
-        <Subtitle color="text-biscuits-brown-200" text="Newsletter" />
-        <Button text="SIGN UP" link="http://eepurl.com/iaWAc5" />
+    <div className="px-10">
+      <Title text="Support Us" food="croissant" color="gray" />
+      <div className="grid grid-cols-2 items-center justify-center py-14">
+        <Image src={standing} alt="standing" />
```
Comment (qhgill) [inline src/components/support/Support.jsx]: move mascot closer to the text

**Comment by xwRongx** [comment]
image

**Review by seanquiambao — APPROVED** [review]

**Pull request merged**